distributed/cli/tests/test_dask_remote.py

```diff
@@ -0,0 +1,7 @@
+from distributed.utils_test import popen
+
+
+def test_dask_remote():
+    with popen(['dask-remote', '--port', '9999']) as remote:
```

Comment:
I would probably just skip this to be honest.  Or I would run the `RemoteClient` object in a normal, non-popen test using `@gen_cluster` and a scheduler.  I find the popen tests to be incredibly finicky.